src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php

```diff
@@ -212,3 +211,54 @@ private function getCompiledContainer()
         return $this->container;
     }
 }
+
+class Router implements RouterInterface
+{
+    private $context;
+
+    public function setContext(RequestContext $context): void
+    {
+        $this->context = $context;
+    }
+
+    public function getContext(): RequestContext
+    {
+        return $this->context;
+    }
+
+    public function getRouteCollection(): RouteCollection
+    {
+        return new RouteCollection();
+    }
+
+    public function generate(string $name, array $parameters = [], int $referenceType = self::ABSOLUTE_PATH): string
+    {
+    }
+
+    public function match(string $pathinfo): array
+    {
+        return [];
+    }
+}
+
+class InMemoryProfilerStorage implements ProfilerStorageInterface
```

Comment:
Isn't this a `NullProfilerStorage`? After all, it does not store in memory. In fact, it does not store anything at all.

Reply:
valid point, fixed